hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/NetworkFeature.java

```diff
@@ -0,0 +1,61 @@
+package com.hedera.mirror.test.e2e.acceptance.steps;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.hedera.mirror.test.e2e.acceptance.client.MirrorNodeClient;
+import com.hedera.mirror.test.e2e.acceptance.props.MirrorNetworkStake;
+
+@Data
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class NetworkFeature {
+
+    private final MirrorNodeClient mirrorClient;
+
+    private MirrorNetworkStake networkStake;
+
+    @When("I query the network stake")
+    public void verifyNetworkStake() {
+        networkStake = mirrorClient.getNetworkStake();
+        assertNotNull(networkStake);
+    }
+
+    @Then("the mirror node REST API returns the network stake")
+    public void verifyNetworkStakeResponse() {
+        assertThat(networkStake.getMaxStakingRewardRatePerHbar()).isPositive();
+        assertThat(networkStake.getNodeRewardFeeFraction()).isGreaterThanOrEqualTo(0);
```

Comment:
Same for other fraction:
```suggestion
        assertThat(networkStake.getNodeRewardFeeFraction()).isBetween(0.0, 1.0);
```

Reply:
Sounds good, I have switched the fraction assertions to `isBetween`.